packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -51,6 +48,13 @@ const ResetPasswordWithRecoveryKeyVerified = ({
     navigate('/settings', { replace: true });
   };
 
+  const [isSync, setIsSync] = useState<boolean>();
+  useEffect(() => {
+    (async () => {
+      setIsSync(await relier.isSync());
```

Comment:
Just thinking outloud since I think we have a ticket filed to make this synchronous. I wonder if we can do that when we combine the integrations and reliers, looks like `clientInfo` is just:

```
export interface RelierClientInfo {
  clientId: string | undefined;
  imageUri: string | undefined;
  serviceName: string | undefined;
  redirectUri: string | undefined;
  trusted: boolean | undefined;
}
```

And I believe if `clientId` is present, it's not going to be sync since that'd be oauth. I _think_ we can base it on the `context` param and `service` param. 🤔